sycl/test-e2e/Plugin/level_zero_sub_sub_device.cpp

```diff
@@ -1,15 +1,18 @@
 // REQUIRES:  gpu-intel-pvc, level_zero
+// REQUIRES: aspect-ext_intel_device_id
```

Comment:
Why do we need that? We already have `REQUIRES: gpu-intel-pvc`.

Reply:
True. I was keeping the IsPVC() test the same for all three of these and it retrieves the device_id.